### ISSUE TITLE
Refine walkthrough process & improve Next Steps

### ISSUE THREAD

**Issue by jamonholmgren** [infinitered/ignite-ir-next#31]
During the walkthrough process, we ask a series of questions, but they're not very well thought through at this point. Let's focus the walkthrough on being a beginner-friendly selection of options.

In addition, let's build out a much better / nicer Next Steps screen once the project is done generating. Include:

• What was generated?
• How do I learn about the structure & tools available in the generated app?
• What are my next steps?
• What plugins are available to add?

**Comment by jamonholmgren** [comment]
This PR is part of this initiative.

https://github.com/infinitered/ignite-ir-next/pull/55

**Comment by jamonholmgren** [comment]
Another PR here #56 .

**Comment by jamonholmgren** [comment]
Going to call this reasonably okay for now. We should continue to work on it.
